eclipse-jarsigner-plugin not available

Builds are currently failing:
```
[ERROR] Plugin org.eclipse.cbi.maven.plugins:eclipse-jarsigner-plugin:1.1.3 or one of its dependencies could not be resolved: Failure to find org.eclipse.cbi.maven.plugins:eclipse-jarsigner-plugin:jar:1.1.3 in https://repo.eclipse.org/content/repositories/maven_central/ was cached in the local repository, resolution will not be reattempted until the update interval of eclipse.maven.central.mirror has elapsed or updates are forced -> [Help 1]
```

Maven Central does not have that plugin: https://search.maven.org/search?q=eclipse-jarsigner-plugin

--- a/maven/org.eclipse.emf.mwe2.parent/pom.xml
+++ b/maven/org.eclipse.emf.mwe2.parent/pom.xml
@@ -245,8 +245,10 @@
 		<profile>
 			<id>cbi-sign</id>
 			<activation>
-				<activeByDefault>true</activeByDefault>
-				<!-- TODO move cbi signing plugin to this profile <property> <name>sign.skip</name> <value>false</value> </property> -->
+				<property>
+					<name>sign.skip</name>
+					<value>false</value>
+				</property>
 			</activation>
 			<pluginRepositories>
 				<pluginRepository>

